package.json

```diff
@@ -28,6 +28,9 @@
         "test:debug": "ts-scripts test --debug asset --",
         "test:watch": "ts-scripts test --watch asset --"
     },
+    "dependencies": {
+        "node-gyp": "9.4.1"
```

Comment:
Did you add this explicitly or did something else do it?

Reply:
The order of properties changes when you run a bump command.